nuget/PlanGrid.Api.nuspec

```diff
@@ -0,0 +1,15 @@
+<package>
+  <metadata>
+    <id>PlanGrid.Api</id>
+    <version>0.0.1</version>
+    <authors>kirk</authors>
+    <owners>kirk</owners>
+    <projectUrl>https://github.com/plangrid/plangrid-api-net</projectUrl>
+    <licenseUrl>http://www.opensource.org/licenses/mit-license.php</licenseUrl>
```

Comment:
I'll let @marchello2000 chime in on how we want to handle file headers vis a vis an open-source license.